library/src/main/java/com/vijay/jsonwizard/validators/textinputlayout/LengthValidator.java

```diff
@@ -0,0 +1,32 @@
+package com.vijay.jsonwizard.validators.textinputlayout;
+
+
+/**
```

Comment:
This code comes from an external project. We should include here the license header and the attribution. Something like this.

 

> Copyright 2014 rengwuxian
> 
> Licensed under the Apache License, Version 2.0 (the "License");
> you may not use this file except in compliance with the License.
> You may obtain a copy of the License at
> 
>    http://www.apache.org/licenses/LICENSE-2.0
> 
> Unless required by applicable law or agreed to in writing, software
> distributed under the License is distributed on an "AS IS" BASIS,
> WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> See the License for the specific language governing permissions and
> limitations under the License.
>
> Extracted from: com.rengwuxian.materialedittext.validation.edittext.LengthValidator